pegasus/helpers/form_helpers.rb

```diff
@@ -195,6 +199,7 @@ def update_form(kind, secret, data)
   form[:reviewed_at] = nil
   form[:reviewed_by] = nil
   form[:reviewed_ip] = nil
+  form[:hashed_email] = Digest::MD5.hexdigest(form[:email]) if data.key?(:email_s)
```

Comment:
Here we're grabbing from a different reference from the empty check @ end of this line, could this reference `:email_s` or empty check on the `form[:email]` data itself? May be misunderstanding the `data[:email_s]` / `form[:email]` relationship